source/grid/grid_out.cc

```diff
@@ -4228,6 +4227,7 @@ namespace internal
       {}
     };
 
+    using LineList = std::list<LineEntry>;
```

Comment:
Our usual convention is to avoid these sort of convenience typedefs since they make it more difficult to read code.